packages/im-components/src/organisms/filmList/FilmList.js

```diff
@@ -44,16 +49,24 @@ const renderMovie = (
   label,
   showDetails,
   isDetailsRightAligned,
-  linkTo
+  application
 ) => {
   const filmId = FilmReader.id(filmDetails);
   const filmTitle = FilmReader.title(filmDetails);
   const filmRating = FilmReader.rating(filmDetails);
   const filmGenre = FilmReader.genre(filmDetails);
   const filmImgSrc = FilmReader.thumbnail(filmDetails);
 
+  let filmDetailsRoute;
+  if (application === APPS.OTT) {
+    filmDetailsRoute = getRoute(APPS.OTT, VIEWS.FILMDETAILS, { filmId });
```

Comment:
why the if check? Pass application directly to getRoute.

Reply:
This is a generic utility.. can be used for both studio and ott routes